src/main/java/org/folio/rest/impl/RtacResourceImpl.java

```diff
@@ -205,6 +219,15 @@ private void assignDueDate(Holding holding, JsonObject responseBody) {
     }
   }
 
+  private String encode(String value) {
```

Comment:
I see you put the same method in Utils.java, so this should be removed.

Reply:
That's a unit test Utils. There was an existing class there, so I used it. It also behaves differently when there is an exception. (that should never happen) due to the JVM inexplicably missing UTF-8 support.

Reply:
Gotcha. This is well-cleaned up now and much easier to read!

Reply:
But to your point, I was thinking about creating a Utils in `src/main`. However, for just this one method, it seemed like overkill. If this module grows (more REST resources), which I doubt it will, then we can reconsider moving the encode method out of the resource implementation class.